### PR TITLE
Update yew to 0.20 and gloo-worker to 0.2

### PR DESCRIPTION
**Pull request by vbkaisetsu**
Blocking changes: #77

### CODE REVIEW

**Comment by kampersanda** [comment]
@vbkaisetsu

With my M1 Mac, I faced an incident where the model load did not finish. How about in your environment?

FYI: The following shows the output logs of my terminal command.

スクリーンショット 2023-01-27 12 46 20

**Comment by vbkaisetsu** [comment]
How is the console on the web browser?

**Comment by vbkaisetsu** [comment]
@kampersanda ↑

**Comment by kampersanda** [comment]
@vbkaisetsu Oh, it seems that `vaporetto_worker.js` failed to load.

スクリーンショット 2023-01-27 12 59 59

How should I do to resolve it?

**Comment by vbkaisetsu** [comment]
@kampersanda Fixed to use the relative path in the `spawn()` function.

**Comment by kampersanda** [comment]
@vbkaisetsu Thanks! It works. Then, wait for my review.

File: examples/wasm/Cargo.toml [hunk 1]
```diff
@@ -4,13 +4,19 @@ version = "0.1.0"
 edition = "2021"
 
 [dependencies]
-bincode = "2.0.0-rc.1"  # MIT
-once_cell = "1.12.0"  # MIT or Apache-2.0
+bincode = "2.0.0-rc.2"  # MIT
+gloo-worker = "0.2.1"  # MIT or Apache-2.0
+once_cell = "1.17.0"  # MIT or Apache-2.0
```
Comment (kampersanda) [inline examples/wasm/Cargo.toml:9]: Is this specification needed?

**Review by kampersanda — COMMENTED** [review]
I am not familiar with yew and gloo_worker so I am not confident in my review; however, I could understand this update while reading the official documentation.

Btw, do you know any good documentation or examples on gloo? There is few documentation.

File: examples/wasm/Cargo.toml (hunk 1 above)
Comment (vbkaisetsu) [reply examples/wasm/Cargo.toml]: No. fixed.

**Comment by vbkaisetsu** [comment]
This is an example of gloo-worker: https://github.com/rustwasm/gloo/tree/master/examples/markdown

**Review by kampersanda — APPROVED** [review]

**Pull request merged**